CMakeLists.txt

```diff
@@ -94,9 +94,22 @@ target_compile_definitions(${TARGET_NAME}
         JUCE_USE_CURL=0
         JUCE_APPLICATION_NAME_STRING="$<TARGET_PROPERTY:${PROJECT_NAME},JUCE_PRODUCT_NAME>"
         JUCE_APPLICATION_VERSION_STRING="$<TARGET_PROPERTY:${PROJECT_NAME},JUCE_VERSION>"
+)
+
+if(MSVC) 
+    target_compile_definitions(${TARGET_NAME}
+        PRIVATE
+              GIT_COMMIT_HASH=\"${GIT_COMMIT_HASH}\"
+             -DGITHUB_REPO_URL="https://github.com/sufst/vcu-gui"
```

Comment:
This one is also susceptible to the same problem, the `-D` should be removed here.

Reply:
Same here.